client/hello.html

```diff
@@ -1,6 +1,11 @@
+<!DOCTYPE html>
 <html>
-  <body>
-    <h1>Hello Echo Chamber</h1>
+<head>
+    <title>Echo Chamber Dummy Dashboard</title>
```

Comment:
this updates the title for the plug in? or it's the header on the page?

Reply:
it should be the header of the page.